includes/forms/functions.php

```diff
@@ -193,6 +193,11 @@ function give_send_back_to_checkout( $args = [] ) {
 	// Set the $level_id.
 	if ( isset( $_POST['give-price-id'] ) ) {
 		$defaults['level-id'] = sanitize_text_field( $_POST['give-price-id'] );
```

Comment:
@ravinderk would this not select the `custom` level id? I remember that working during my testing (I may be misremembering).

Reply:
@kjohnson this give me correct price id and redirect me to correct url.

Reply:
Ok, I think I see now. This redirects correctly, but does not update the price id *after* the redirect.